pages/data/utils.ts

```diff
@@ -0,0 +1,377 @@
+import { Entry } from "contentful";
```

Comment:
Please remove this file. All the content in this file is now found elsewhere. Specifically the methods needed for fetching algolia facets are in utils/algolia.ts while things like extractExtension, searchQueryReplacements, highlight matches, saveForm, etc... are in utils/utils.ts